src/nimble.nim.cfg

```diff
@@ -4,3 +4,4 @@
 --path:"./vendor/nim"
 -d:ssl
 -d:nimcore # Enable 'gorge' in Nim's VM. See https://github.com/nim-lang/Nim/issues/8096
+-d:nimOldCaseObjects
```

Comment:
Why is this needed now?

Reply:
when i built nimble manually, without koch, it crashed without this flag.

Reply:
You are using a really old version of nimble. Need to get the latest codebase.

Reply:
Ah, forgot to `pull`